maintainers/maintainer-list.nix

```
@@ -5222,6 +5222,10 @@
     githubId = 10626;
     name = "Andreas Wagner";
   };
+  lrworth = {
+    email = "luke@worth.id.au";
+    name = "Luke Worth";
```

Please add your github name and ID.

---

https://github.com/NixOS/nixpkgs/blob/2520209f42aa409d33400801aaa2a17e969a03ff/maintainers/maintainer-list.nix#L8-L10

---

I'd prefer not to list those if they are truly optional.

---

```suggestion
    name = "Luke Worth";
    github = "lrworth";
    githubId = 758855;
```

There is no reason to not add them because this is public information anyway.

---

Disagree: if they're optional in the file, and a contributor says "are these really optional? I'd rather not put them in" I don't think digging that information up is a very nice thing to do.

---

It wouldn't be the first time I would need to dig up some maintainer and adding this infos just makes this less painful and it is not like I could not find these **public** information anyway.

> digging 

I literally copied his username from **this** PR and did one api request to github which just gives publicly  accessible information.

I don't see any reason for the fuzz.

---

In the interest of having this merged, I've removed myself as a maintainer.

---

Adding new packages without maintainer is a merge blocker.